### PR TITLE
Added meta apis for ONDC and UMTC

### PR DESCRIPTION
**Pull request by em-vivek**

### CODE REVIEW

File: src/mobility/mobility.controller.ts [hunk 1]
```diff
@@ -56,4 +56,15 @@ export class MobilityController {
     rating(@Body() mobilityRequest: MobilityRequestDto) {
         return this.service.get(mobilityRequest.context.domain).rating(mobilityRequest);
     }
+
+    @Post("/get_cancellation_reasons")
+    cancellation_reasons(@Body() mobilityRequest: MobilityRequestDto) {
```
Comment (nikhil2128) [inline src/mobility/mobility.controller.ts:61]: Use camel casing for function name. Like cancellationReasons

File: src/mobility/mobility.controller.ts [hunk 2]
```diff
@@ -56,4 +56,15 @@ export class MobilityController {
...
+        return this.service.get(mobilityRequest.context.domain).cancellation_reasons(mobilityRequest);
+    }
+
+    @Post("/get_rating_categories")
+    rating_categories(@Body() mobilityRequest: MobilityRequestDto) {
```
Comment (nikhil2128) [inline src/mobility/mobility.controller.ts:66]: same as above

File: src/ondc/ondc-api.service.ts [hunk 3]
```diff
@@ -56,4 +58,11 @@ export class OndcApiService implements MobilityApiInterface {
     rating = function (ratingRequest: RatingRequestDto) {
         return ratingResponse;
     };
+    cancellation_reasons = function () {
+        return cancellationResponse
+    }
+    rating_categories = function () {
+        return ratingCategoriesResponse
```
Comment (nikhil2128) [inline src/ondc/ondc-api.service.ts:65]: same as above

File: src/umtc/umtc-api.service.ts [hunk 4]
```diff
@@ -55,4 +56,10 @@ export class UmtcApiService implements MobilityApiInterface {
     support = function (supportRequest: SupportRequestDto) {
         return supportResponse;
     };
+    cancellation_reasons = function () {
+        return cancellationResponse
+    }
+    rating_categories = function () {
```
Comment (nikhil2128) [inline src/umtc/umtc-api.service.ts:62]: same as above

**Review by nikhil2128 — CHANGES_REQUESTED** [review]

**Review by nikhil2128 — APPROVED** [review]

**Review by prasad-takale-eminds — APPROVED** [review]

**Pull request merged**